src/borg/repository.py

```diff
@@ -1482,6 +1483,29 @@ def iter_objects(self, segment, offset=0, include_data=False, read_data=True):
             fd.seek(offset)
             header = fd.read(self.header_fmt.size)
 
+    def recover_segment_chunk(self, d, try_next_chunk=True, pedantic=False):
+        """
+        Returns a tuple of chunk data (or None) and length of bytes to skip forward
+
+        If try_next_chunk is True, then when the CRC mismatches, it will seek past the purported size and check the CRC of the next chunk. If that is correct, or exactly EOF, it will be returned with a length of both the corrupt chunk and the next one being returned.
```

Comment:
this line is way too long and triggers the pep8 checker, see travis-ci output.

Reply:
PEP8 is stupid, and the other comments don't follow this rule, so I'd rather not bother tbh...

Reply:
our pep8 checker is configured to ignore some stuff and has high limits on line length, but that line is beyond with no good reason.

Reply:
I've fixed that for you.